Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -25,4 +26,12 @@ contexts:
     - match: ''
       push:
         - meta_scope: meta.content.multimarkdown
-        - include: scope:text.html.markdown
+        - match: ''
+          embed: scope:text.html.markdown
+          escape: $(?!$)  # should never match
```

Comment:
Why an embed here that never exits? This will add overhead where we keep track of a state-based Oniguruma regex that is run on every line, before the regular matching is done.

Reply:
It seemed like the proper directive semantically, except that it doesn't have the ability to never exit. I agree that having to run a never-matching pattern for every line is suboptimal, however, especially if it's not using sregex.
I initially went that route because the only other alternative I could think of was to add another nesting level so the markdown context doesn't set out of the context with `meta_scope`, but using a multi-push is much more elegant.